blockchain/README.md

```diff
@@ -1,44 +1,44 @@
 # Blockchain
 
-A Blockchain is a type of **distributed ledger** technology (DLT) that consists of growing list of records, called **blocks**, that are securely linked together using **cryptography**.
+A Blockchain is a type of **distributed ledger** technology (DLT) that consists of a growing list of records, called **blocks**, that are securely linked together using **cryptography**.
 
-Let's breakdown the terminologies in the above definition. We find below terminologies,
+Let's break down the terminologies in the above definition. We find below terminologies,
 
 - Digital Ledger Technology (DLT)
 - Blocks
 - Cryptography
 
 ## Digital Ledger Technology
 
- It is otherwise called as distributed ledger technology. It is simply the opposite of centralized database. Firstly, what is a **ledger**? A ledger is a book or collection of accounts that records account transactions.
+ It is otherwise called distributed ledger technology. It is simply the opposite of a centralized database. Firstly, what is a **ledger**? A ledger is a book or collection of accounts that records account transactions.
 
- *Why is Blockchain addressed as digital ledger if it can record more than account transactions? What other transaction details and information can it hold?*
+ *Why is Blockchain addressed as a digital ledger if it can record more than account transactions? What other transaction details and information can it hold?*
```

Comment:
```suggestion
*Why is Blockchain addressed as a digital ledger if it can record more than account transactions? What other transaction details and information can it hold?*
```